Documentation/building/cross-building.md

```diff
@@ -1,7 +1,23 @@
-Cross Compilation for ARM
-=========================
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
+
+Requirements
+------------
+
+You need a Debian based host and the following packages needs to be installed:
+
+    ben@ubuntu ~/git/coreclr/ $ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
```

Comment:
You need those for cross compilation, so they apply to both generating the rootfs and compiling.